src/Raven.Studio/wwwroot/App/views/database/settings/customAnalyzers.html

```diff
@@ -0,0 +1,46 @@
+<div class="destinations flex-vertical absolute-fill content-margin">
```

Comment:
can we remove destinations class? I bet we have same issues in custom sorters page - please check if it doesn't break anything.

Reply:
can remove, removed.